### ISSUE TITLE
nodejs v 4

### ISSUE THREAD

**Issue by Valonix** [scotch-io/mean-machine-code#31]
Hello. I want to buy ur book. But I want to know, right now nodejs is v4.1 and he ea6 from box. It is not problem if I start to learn with old version.?

**Comment by josephkandi** [comment]
@Valonix I don't think you need to wait, the book will still be relevant and it will only be a couple of changes to make your code compatible. I also just started learning a few weeks ago
